packages/reframed/src/reframed.ts

```diff
@@ -111,7 +111,11 @@ async function reframeWithFetch(
 		targetContainer: target,
 	});
 
-	const reframedHtmlResponse = await fetch(reframedSrc);
+	const reframedHtmlResponse = await fetch(reframedSrc, {
```

Comment:
for soft-navigation

Reply:
referring to "fragment mode" in `reframed` feels a bit like we've got a leaky abstraction. perhaps it would be better to add a `headers?` option to `reframed(options)` instead? then the headers could be defined entirely within the `web-fragments` package